src/Interactive/Features/Interactive/Core/InteractiveHost.RemoteService.cs

```diff
@@ -61,6 +62,7 @@ internal void HookAutoRestartEvent()
                         if (Interlocked.Exchange(ref processExitHandling, ProcessExitHandled) == ProcessExitHooked)
                         {
                             Process.Exited -= localHandler;
+                            _currentLocalHandler = null;
```

Comment:
Why the local method now vs. just a regular good old fashioned method? I realize it was nice when it was localized to this, but that ship has sailed.